src/responsibleai/rai_analyse/_score_card/_rai_insight_data.py

```diff
@@ -397,13 +411,22 @@ def get_data_explorer_data(self):
         de_data = []
         y_test = self.data.get_y_test()
         y_predict = self.data.get_y_pred()
-        for f in self.config["DataExplorer"]["features"]:
-            label_list, new_labels = self.get_binning_information(f)
+
+
+        for feature in self.config["DataExplorer"]["features"]:
+
+            if feature not in self.data.get_raiinsight().error_analysis._analyzer.feature_names:
```

Comment:
Can you check the feature names against "self.data.get_raiinsight().test.drop([columns=[self.data.get_raiinsight().target_column]).columns" instead. 

Looks weird we would use error_analysis to derive the feature names. Also, does it work when user doesn't configure EA.

Reply:
get_filtered_dataset relies on the same which is why I concluded it's fine (for now)

I'll switch to what you're suggesting, but I'll switch over that one, too.